html/css/main.css

```diff
@@ -315,14 +315,14 @@ body {
 
 .phone-notification-container {
     position: absolute;
-    height: 5.5vh;
+    height: 7.0vh;
```

Comment:
@wanderrer what's the reason for the css edits? Were they to fix something?

Reply:
Changing this css is outside of the scope of this PR